flake.nix

```
@@ -0,0 +1,27 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs = {nixpkgs, ...}: let
+    system = "x86_64-linux";
+    pkgs = nixpkgs.legacyPackages.${system};
```

question (non-blocking): I used this instead in my own flake:

```suggestion
    pkgs = import nixpkgs { inherit system; };
```

are they doing the same thing?

---

After skimming [this thread](https://discourse.nixos.org/t/using-nixpkgs-legacypackages-system-vs-import/17462), I think that their effect is the same.